frontend/ios/DroidKaigi 2019/Scenes/Session/SessionTableViewCell.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  SessionTableViewCell.swift
+//  DroidKaigi 2019
+//
+//  Created by ShoMasegi on 2019/01/22.
+//
+
+import UIKit
+import ios_combined
+import SnapKit
+
+class SessionTableViewCell: UITableViewCell, Reusable {
+
+    var session: Session? {
+        didSet {
+            guard let session = session else { return }
+            timeAndRoomLabel.text = "\(session.timeInMinutes)min / \(session.room.name)"
```

Comment:
Can you use `session#sumally()`?
https://github.com/DroidKaigi/conference-app-2019/blob/master/model/src/commonMain/kotlin/Session.kt#L39

Reply:
Sorry It contains not needed texts 🙇